crates/next-custom-transforms/src/transforms/server_actions.rs

```diff
@@ -123,6 +123,22 @@ struct ServerActions<C: Comments> {
     private_ctxt: SyntaxContext,
 }
 
+trait IdentCollector {
+    fn collect(&mut self, id: Id, span: Option<Span>);
+}
+
+impl IdentCollector for Vec<Id> {
+    fn collect(&mut self, id: Id, _span: Option<Span>) {
+        self.push(id);
+    }
+}
+
+impl IdentCollector for Vec<(Id, Span)> {
```

Comment:
You can use `Ident` instead.

Reply:
`Ident` has `sym: JsWord`, 'ctxt: SyntaxContext`, and `span: Span` where `Id` is an alias of `(JsWord, SyntaxContext)`.

Reply:
I considered that, but the functions that collect these `Id`s are used for both `exported_idents` and `declared_idents`. For the latter, we don't care about the spans. And then there's also this case where we don't have a complete ident available:

https://github.com/vercel/next.js/blob/e4f5763aa03ae690ff18647000044bd5bb7cf140/crates/next-custom-transforms/src/transforms/server_actions.rs#L2236-L2241

Reply:
@kdy1 More like [that](https://github.com/vercel/next.js/pull/70722)?